package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "passable",
-  "version": "7.5.1",
+  "version": "7.5.2",
```

This is done automatically when merging from next -> relese

```suggestion
  "version": "7.5.1",
```